### DISCUSSION TITLE
Daily Notes Rethink

### DISCUSSION THREAD

**Discussion by ithakaa** [athensresearch/athens#1399] [Design Submissions]
I'm wondering if it's possible to rethink the way daily notes are presented

Let me give a simple example. I love that I get a new note at the start of my work day, makes it easy to start note taking.

What might be even better is to have the previous day's daily note at the mid way down the page as a reference to today's daily note, with smaller font size for yesterday's date.

Given time the daily notes page can be a single page that is easily scrollable backwards in time

I can also see a sliding scale, a simple line with markings and a date, magnified as you scroll, on one side for easy access to notes based on when notes were taken, with some fancy magnify glass effect

**Comment by sawhney17** [comment]
At the moment, this is how it works, with you being able to scroll down while on the daily note page. Is this what you are talking about?

[https://www.loom.com/share/e58aefcd496d474fa70aa04b3d510ee8](https://www.loom.com/share/e58aefcd496d474fa70aa04b3d510ee8)

**Reply by ithakaa** [reply]
I didn't even know this was a thing, excellent !!!
